Az.DevTestLabs2.psm1

```diff
@@ -1790,6 +1810,46 @@ function Get-AzDtlLabVirtualNetworkSubnets {
   end {}
 }
 
+function Set-AzDtlVmCreationResourceGroup {
+  [CmdletBinding()]
+  param(
+    [parameter(Mandatory=$true,HelpMessage="Lab where to update the VmCreationResourceGroup.", ValueFromPipeline=$true)]
+    [ValidateNotNullOrEmpty()]
+    $Lab,
+
+    [parameter(Mandatory=$false,HelpMessage="The resource group in which all new lab virtual machines will be created. To let DevTest Labs manage resource group creation, set this value to null.")]
+    [string] $VmCreationResourceGroupName
+  )
+
+  begin {. BeginPreamble}
+  process {
+    try {
+
+      $Lab = $Lab | Get-AzDtlLab
```

Comment:
We should just make sure that Get-AzDtlLab is using the same API version that you're using below to set back the lab properties...  (if there is a version mismatch, we might end up accidently with overwriting properties)